schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE animals  (
+  id BIGSERIAL PRIMARY KEY NOT NULL,
+  animal_name TEXT NOT NULL,,
+  date_of_birth DATE NOT NULL,
+  escape_attempts INT NOT NULL,
+  neutered BOOLEAN NOT NULL,
+  weigth_kg DECIMAL NOT NULL,
```

Comment:
- [ ] Kindly remove the last comman here too. It does not need since that is the last field you declared in the table. It throws error while trying to create animals table because its presence. I had to remove it to have my way

### Error
![last-comman](https://github.com/amanuelbelete95/vet-clinic-database/assets/26597432/0d27aa95-a500-483b-b11a-f4b406aa5f3f)